### PR TITLE
Added draw action

### LINKED ISSUE #38: Add draw card action type
Add draw card action type in engine and graphics

### PR DESCRIPTION
**Pull request by aleksplast**
- Added draw action class
- Added it's handling into parser and graphics

### CODE REVIEW

File: include/graphics/command.hpp [hunk 1]
```diff
@@ -39,6 +40,7 @@ class Command {
     const std::array<Type, 1> close_window{Type::close_window};
     const std::array<Type, 2> choose_minion{Type::minion, Type::minion};
     const std::array<Type, 2> choose_base{Type::base, Type::base};
+    const std::array<Type, 2> draw_action{Type::draw_action, Type::base};
```
Comment (GauchKirill) [inline include/graphics/command.hpp]: May, types should be "draw_action" and "deck"?

**Review by GauchKirill — COMMENTED** [review]
I have corrected the code

**Comment by GauchKirill** [comment]
@aleksplast, can you check my review?

File: src/parser/parser.cpp [hunk 2]
```diff
@@ -54,6 +54,7 @@ Entity *Parser::parse_minion(std::vector<Entity *> &entities, const Value &item_
 }
 
 Entity *Parser::parse_action(std::vector<Entity *> &entities, const Value &item_value) {
+    std::cout << "Parsing action\n";
```
Comment (aleksplast) [inline src/parser/parser.cpp]: Delete this

Comment (aleksplast) [reply src/parser/parser.cpp]: @GauchKirill Fix this one

Comment (GauchKirill) [reply src/parser/parser.cpp]: Done

**Review by GauchKirill — COMMENTED** [review]
To master?

**Comment by aleksplast** [comment]
> To master?

LGTM

**Pull request merged**